debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleDefaultValueConverter.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.annotation.ThreadSafe;
+import io.debezium.jdbc.JdbcConnection;
+import io.debezium.relational.Column;
+import io.debezium.relational.DefaultValueConverter;
+import io.debezium.relational.ValueConverter;
+import io.debezium.util.Strings;
+
+import oracle.jdbc.OracleTypes;
+import oracle.sql.INTERVALDS;
+import oracle.sql.INTERVALYM;
+import oracle.sql.TIMESTAMP;
+import oracle.sql.TIMESTAMPLTZ;
+import oracle.sql.TIMESTAMPTZ;
+
+/**
+ * @author Chris Cranford
+ */
+@ThreadSafe
+public class OracleDefaultValueConverter implements DefaultValueConverter {
+
+    private static Logger LOGGER = LoggerFactory.getLogger(OracleDefaultValueConverter.class);
+
+    /**
+     * Converts JDBC string representation of a default column value to an object.
+     */
+    @FunctionalInterface
+    private interface DefaultValueMapper {
```

Comment:
We have that type multiple times now. Can we unify it?

Reply:
Yep, so in the commits I'll be adding shortly, I've pulled this functional interface up to the `DefaultValueConverter` contract in core as well.  I've also unified the function signature to match that for Oracle across PostgreSQL and SQL Server connectors.